Assets/Scripts/Framework/Train/TestEngineController.cs

```diff
@@ -0,0 +1,19 @@
+using FishNet.Object;
+using UnityEngine.InputSystem;
+
+namespace DerailedDeliveries.Framework.Train
+{
+    /// <summary>
+    /// Test class responsible for debugging train engine functionality.
+    /// </summary>
+    public class TestEngineController : NetworkBehaviour
+    {
+        private void Update()
+        {
+            if (Keyboard.current.backspaceKey.wasPressedThisFrame) TrainEngine.Instance.ToggleTrainDirection();
+
+            if (Keyboard.current.wKey.wasPressedThisFrame) TrainEngine.Instance.AdjustSpeed(true);
+            if (Keyboard.current.sKey.wasPressedThisFrame) TrainEngine.Instance.AdjustSpeed(false);
```

Comment:
Yup this will indeed be deleted before merge

Reply:
Then it's all good.

Reply:
In my opinion this can stay until we have the train control interactables. Though, I would change the keybinds since some of them conflict with moving the player.

Reply:
Ah yea didn't think of that, in that case we can keep it untill there is an intended way to interact with the engine. I will change the keybinds (and make it configurable)